momento-sdk/src/main/java/PreviewStorageClientDriver.java

```diff
@@ -0,0 +1,89 @@
+import momento.sdk.IPreviewStorageClient;
+import momento.sdk.PreviewStorageClient;
+import momento.sdk.exceptions.ClientSdkException;
+import momento.sdk.responses.storage.control.CreateStoreResponse;
+import momento.sdk.responses.storage.control.ListStoresResponse;
+import momento.sdk.responses.storage.data.GetResponse;
+
+public class PreviewStorageClientDriver {
+  public static void main(String[] args) {
+    // Instantiating the client will be largely the same as compared to cache/topics.
+    IPreviewStorageClient client = new PreviewStorageClient();
+
+    // Create a store
+    CreateStoreResponse createResponse = client.createStore("myStore").join();
+    if (createResponse instanceof CreateStoreResponse.Success) {
+      System.out.println("Store created successfully");
+    } else {
+      // Would inspect for Already exists, etc.
+      System.out.println("Error creating store");
+    }
+
+    // List all stores
+    final ListStoresResponse listResponse = client.listStores().join();
+    if (listResponse instanceof ListStoresResponse.Success) {
+      ListStoresResponse.Success success = (ListStoresResponse.Success) listResponse;
+      System.out.println("Stores:");
+      success.getStores().forEach(store -> System.out.println("- Store: " + store.getName()));
+    } else if (listResponse instanceof ListStoresResponse.Error) {
+      System.out.println(
+          "Error listing stores: " + ((ListStoresResponse.Error) listResponse).getMessage());
+    } else {
+      System.out.println("Unknown response type");
+    }
+
+    // Set a value in the store
+    client.set("myStore", "myKey", "myValue").join();
+
+    // Get the value from the store
+    final GetResponse stringResponse = client.get("myStore", "string").join();
+    printGenericGetResponse(stringResponse);
+
+    final GetResponse doubleResponse = client.get("myStore", "double").join();
+    printGenericGetResponse(doubleResponse);
+
+    if (stringResponse instanceof GetResponse.Success) {
+      GetResponse.Success success = (GetResponse.Success) doubleResponse;
+      System.out.println("Value type for \"double\" was: " + success.getType());
+      System.out.println("Value: " + success.getValueAsDouble());
+
+      // Exception
+      try {
+        System.out.println("Value: " + success.getValueAsString());
+      } catch (ClientSdkException e) {
+        System.out.println(
+            "Error trying to access the value of \"double\" as a string: " + e.getMessage());
+      }
+    } else if (stringResponse instanceof GetResponse.Error) {
+      GetResponse.Error error = (GetResponse.Error) doubleResponse;
```

Comment:
same comment here about using a different var in the condition vs the cast

Reply:
Another typo. Good 👁️